webapp/client/src/components/FormFieldTextInput.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabel from "./FieldLabel";
+import FormFieldSeparatedField from "./FormFieldSeparatedField";
+
+function FormFieldTextInput({
+  fieldName,
+  fieldId,
+  value,
+  required,
+  autofocus,
+  label,
+  maxLength,
+  setMaxLength,
+  details,
+  errors,
+}) {
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+      }}
+      labelComponent={<FieldLabel {...{ label, fieldId, details }} />}
+      render={() => (
+        <input
+          type="text"
+          id={fieldId}
+          name={fieldName}
+          defaultValue={value}
+          maxLength={setMaxLength ? maxLength : null}
```

Comment:
couldn't we just use the maxLenght prop here and only set it if that has a value?

I think that this should work:
`maxLength={maxLength}`

Reply:
If we still want to be able to adapt the width, we could do that explicitly by using a maxLength and a maxWidth.